mozci/data/sources/treeherder/__init__.py

```diff
@@ -182,3 +185,16 @@ def run_push_tasks_classifications(self, branch, rev):
                 ]
 
         return result
+
+    def run_push_test_groups(self, branch, rev):
+        if Push:
+            push = Push.objects.get(repository__name=branch, revision=rev)
+            result = get_group_results(push)
+
+            return result
+
+        logger.trace(
+            "Unable to reach Treeherder as a datasource because the Push model "
+            "was not available to import."
+        )
+        return {}
```

Comment:
I'd do as we do for the other contract implementations in this class, `raise ContractNotFilled` if we can't import the model

Reply:
Shoot, my bad.  OK, fixed this.